config/initializers/secret_token.rb

```diff
@@ -1,7 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-# Your secret key for verifying the integrity of signed cookies.
+# Your secret key is used for verifying the integrity of signed cookies.
 # If you change this key, all old signed cookies will become invalid!
+
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
 Lims3::Application.config.secret_token = 'a4894c186c085cabe496af731838f541fb664763ed0900e9cee471790f0dd734965817bf10d36401ff7d72e86d62146a96844dc356b809329be2f71aca48fdec'
```

Comment:
Thanks, likely it should, but I had left it this way primarily because we should get rid of this file entirely and use secrets.yml instead. I have a brief note on this in this PR and the rails40 PR, but didn't want to change security procedure without your OK.